src/variables.css

```diff
@@ -1,38 +1,37 @@
-
 :root {
 	/* Colors */
-	--fg: var(--gray-12); /* Text. */
-	--muted-fg: var(--gray-10); /* Secondary text color. Will be
+	--fg: light-dark(var(--gray-12), var(--gray-0)); /* Text. */
+	--muted-fg: light-dark(var(--gray-10), var(--gray-2)); /* Secondary text color. Will be
 		used with a background of --c-bg and --c-bg-2 -- check contrast! */
-	--faded-fg: var(--gray-6); /* Disabled text color. */
+	--faded-fg: light-dark(var(--gray-6), var(--gray-7)); /* Disabled text color. */
 	--graphical-fg: var(--plain-graphical-fg); /* Graphical elements. Will not
 		be used as a text color. */
 	
-	--plain-fg: var(--blue-10);
-	--info-fg: var(--blue-11);
-	--ok-fg: var(--green-11);
-	--bad-fg: var(--red-11);
-	--warn-fg: var(--yellow-11);
-
-	--plain-graphical-fg: var(--gray-6);
-	--info-graphical-fg: var(--blue-6);
-	--ok-graphical-fg: var(--green-6);
-	--bad-graphical-fg: var(--red-6);
-	--warn-graphical-fg: var(--yellow-6);
+	--plain-fg: light-dark(var(--blue-10), var(--blue-2));
+	--info-fg: light-dark(var(--blue-11), var(--blue-2));
+	--ok-fg: light-dark(var(--green-11), var(--green-2));
+	--bad-fg: light-dark(var(--red-11), var(--red-2));
+	--warn-fg: light-dark(var(--yellow-11), var(--yellow-2));
+
+	--plain-graphical-fg: light-dark(var(--gray-6), var(--blue-6));
+	--info-graphical-fg: light-dark(var(--blue-6), var(--blue-6));
+	--ok-graphical-fg: light-dark(var(--green-6), var(--green-6));
+	--bad-graphical-fg: light-dark(var(--red-6), var(--red-6));
+	--warn-graphical-fg: light-dark(var(--yellow-6), var(--yellow-6));
 	
-	--bg: var(--gray-0); /* Page background. */
-	--box-bg: var(--plain-bg); /* Background for blocks: cards, admonitions etc. */
-	--interactive-bg: var(--gray-4); /* Background for interactive elements */
+	--bg: light-dark(var(--gray-0), var(--gray-12)); /* Page background. */
+	--box-bg: light-dark(var(--plain-bg), var(--gray-10)); /* Background for blocks: cards, admonitions etc. */
```

Comment:
@dz4k Should the dark value for `--box-bg` be `--plain-bg`? (aka `--gray-11` instead of `--gray-10`)

Reply:
I think so, yeah